boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280-memory_map.dtsi

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/dt-bindings/memory-attr/memory-attr.h>
+
+/ {
+	reserved-memory {
+		cpuapp_ram0x_region: memory@2f011000 {
```

Comment:
cpuapp_ram0x_region and cpurad_ram0x_region were recently swapped in the memory layout design to make creating cpuapp overlay files simpler. Ref http://int-ncs-doc.nordicsemi.no/main/html/internal/sw_arch/memory_maps/nrf9230.html#general-purpose-shared-ram-ram0x

Reply:
Thank you, I have updated this.